apps/api/core/logs.py

```
@@ -2,6 +1,92 @@
+
+    # Configure stream handler for stdout
+    stream_handler = logging.StreamHandler(sys.stdout)
+    
+    # Use the custom JSON formatter
+    # Example format: %(asctime)s %(levelname)s %(name)s %(module)s %(funcName)s %(lineno)d %(message)s
+    # These will be automatically picked up by JsonFormatter if present in the log record's extra fields
+    # or can be added to the format_string if needed for non-extra fields.
+    formatter = CustomJsonFormatter(
+        fmt="%(asctime)s %(levelname)s %(name)s %(message)s",
+        rename_fields={"levelname": "level", "asctime": "timestamp"},
+        datefmt="%Y-%m-%dT%H:%M:%S.%fZ" # ISO 8601 format
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)